cli/README.md

```diff
@@ -10,275 +10,4 @@ cargo run --package divviup-cli -- help
 
 If you run into problems with the `aws-lc-rs` dependency, you can try building with `--no-default-features --features common,ring`.
 
-Or you can download a binary for your OS and host architecture from our [releases](https://github.com/divviup/divviup-api/releases).
```

Comment:
I think we should keep this line (it's the second clause after the "either" above).

Reply:
fixed